modules/helm-creating-a-custom-helm-chart-on-openshift.adoc

```diff
@@ -28,10 +28,16 @@ $ cd redhat-helm-charts/alpha/nodejs-ex-k/
 +
 [source,yaml]
 ----
-apiVersion: v2
-name: Node.js
-description: A Helm chart for OpenShift
+apiVersion: v2 <1>
+name: nodejs-ex-k <2>
+description: A Helm chart for Openshift <3>
```

Comment:
This changed "OpenShift" to "Openshift". It should stay "OpenShift".

Reply:
Thank you for the review! 
Fixed.